ansible/roles/netbox_ansible/tasks/setup_tasks.yaml

```
@@ -0,0 +1,45 @@
+  loop_control:
+    label: "{{ item.name }}"
+- name: "TASK 20: SETUP MANUFACTURERS"
+  netbox.netbox.netbox_manufacturer:
+    netbox_url: "{{ netbox_url }}"
+    netbox_token: "{{ netbox_token }}"
+    data:
+      name: "{{ manufacturer }}"
+  loop: "{{ manufacturers }}"
+  loop_control:
+    loop_var: manufacturer
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)